shared/elements/gaia_grid_rs/style.css

```diff
@@ -409,3 +409,35 @@ instead of the context menu.
 :-moz-dir(rtl) gaia-grid-rs {
   direction: ltr;
 }
+
+
+gaia-grid-rs {
+  display: block;
+  height: 20rem !important;
```

Comment:
Maybe for such temporary solutions it makes sense to add appropriate comment, or even TODO: comment?